### PR TITLE
[Enhancement] Create project tags for 2024 Mobility Annual Plan (MAP)

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/12859

## Testing

local 

**Steps to test:**

On a project summary page, see the new 2024 tags available in the tags dropdown. 

How do we feel about the width of the tag input? I think it could be wider, especially since the tags are wordy.....

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Review by roseeichelmann — APPROVED** [review]
The tags look good to me, personally i think the width is fine!

**Review by mddilley — APPROVED** [review]
I tested these tags out locally and see all of them in the UI. Migrations looks great. 🙏 🚢

File: moped-database/migrations/1689366577635_add_map_2024_tags/up.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO public.moped_tags ("type", name, slug) values
+        ('Work Plan', 'MAP Signals 2024', 'map_signals2024'),
+        ('Work Plan', 'MAP Transit Enhancement 2024','map_transitenhancement2024'),
+        ('Work Plan', 'MAP Transit Speed and Reliability (CapMetro ILA) 2024', 'map-transit-speed-and-reliability-capmetro-ila-2024'),
+        ('Work Plan', 'MAP Vision Zero - Major Safety 2024', 'map-vision-zero-major-safety-2024'),
+        ('Work Plan', 'MAP Vision Zero - Ped Safety 2024', 'map-vision-zero-ped-safety-2024'),
+        ('Work Plan', 'MAP Vision Zero - Rapid Response/Small Scale 2024', 'map-vision-zero-rapid-response-small-scale-2024'),
```
[the remark this answers is not in the document]
Comment (johnclary) [reply moped-database/migrations/1689366577635_add_map_2024_tags/up.sql:7]: i agree i want the input to be wider. i will make an issue 🙏

**Review by frankhereford — APPROVED** [review]
Thank you! 🏷️

**Pull request merged**